test.js

```diff
@@ -636,12 +636,97 @@ function runTests(baseopts) {
           });
         });
     });
+    it('should detect unlink unfazed by in other directory watching a second file that does not exist', function(done) {
```

Comment:
little phrasing tweaks

`should detect unlink while watching a non-existent second file in another directory`